com/win32comext/axdebug/codecontainer.py

```diff
@@ -253,11 +252,10 @@ def GetName(self, dnt):
 
 
 if __name__ == "__main__":
-    sys.path.append(".")
-    import ttest
+    from Test import ttest
```

Comment:
I tested running this script from both `pywin32\com\win32comext\axdebug` and `pywin32\com\win32comext\axdebug\Test`. With this change, it finds the module in both cases.